networking/questions.py

```diff
@@ -27,10 +32,9 @@ def IngressQuestions(ask=True):
     #     ]
     #     r = questionary.prompt(questions)
     #     result.update(r)
-    return result
 
 
-def EgressQuestions(ask=True):
+"""def EgressQuestions(ask=True):
```

Comment:
@slyons we are not asking this anymore?

Reply:
It was already commented out: https://github.com/SimpleDataLabsInc/pre-install-script/blob/3b5764e8b73da767901ea3538152818cd5008edb/networking/questions.py#L56

Looks like we're not using it in the helm command either